deeppavlov/agents/default_agent/default_agent.py

```diff
@@ -47,7 +48,7 @@ class DefaultAgent(Agent):
         skills_processor: Initiated agent processor.
         skills_filter: Initiated agent filter.
     """
-    def __init__(self, skills: List[Skill], skills_processor: Optional[Processor]=None,
+    def __init__(self, skills: Union[List[Skill], List[Component]], skills_processor: Optional[Processor]=None,
```

Comment:
```suggestion
    def __init__(self, skills: List[Component], skills_processor: Optional[Processor]=None,
```
`Skill` is a `Component`, right?

Reply:
Fixed